content/02.introduction.md

```diff
@@ -10,66 +10,62 @@ One week later, 4593 confirmed cases had been identified, spanning not only Asia
 On March 11, 2020, WHO formally classified the situation as a pandemic [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200311-sitrep-51-covid-19.pdf].
 On April 4, 2020, the WHO reported that the global number of confirmed cases had surpassed one million [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200404-sitrep-75-covid-19.pdf?sfvrsn=99251b2b_4].
 At this time, over 92,000 deaths had been reported due to COVID-19 worldwide.
+New information is emerging daily about the impact of the disease around the world.
 
-[Note: Maybe add a graph here, update as new reports come out.]
-
-COVID-19 is caused by the newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2).
-SARS-CoV-2 is a coronavirus, a family of RNA viruses known to cause respiratory and intestinal infections in humans and other species.
+COVID-19 is caused by the newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which was initially called 2019 novel Coronavirus (19-nCov).
+SARS-CoV-2 belongs to coronavirus, a family of RNA viruses known to cause respiratory and intestinal infections in humans and other species.
```

Comment:
```suggestion
SARS-CoV-2 belongs to the family of coronaviruses, which are RNA viruses known to cause respiratory and intestinal infections in humans and other species.
```

Reply:
I think i would say something like "Coronavirus is a genus of Coronaviridae that is known to ...." The current sentence implies to me, at least, that coronaviruses have this attribute of causing respiratory and intestinal infections as opposed to Toroviruses, which I don't think is true